plugins/module_utils/network/common/utils.py

```diff
@@ -685,11 +685,11 @@ def __call__(self, value, variables=None, fail_on_undefined=True):
                 return None
             raise
 
-        if value:
-            try:
-                return ast.literal_eval(value)
-            except Exception:
-                return str(value)
+        if value is not None:
```

Comment:
@AAYUSH2091 we generally put try and exception block to handle corner cases so i dont think removing this would be a good option here, so you might wanna revert it back if thats not something blocking you but we cant just refactor the code this way, this might open other potential problems/errors for us.